docs/UserGuide.md

```diff
@@ -44,179 +44,246 @@ faster than traditional GUI apps.
   ignored.<br>
   e.g. if the command specifies `listcustomers 123`, it will be interpreted as `listcustomers`.
 
+* Parameters expecting dates must be given in these formats: `DD/MM/YYYY` or `YYYY-MM-DD` or `MMM DD YYYY`,<br>
+  e.g. `31/03/2021` or `2021-03-31` or `Mar 31 2021` where `Mar` is case-sensitive.
+
+* Parameters for dates will accept any day given in the range [1..31].
+  If the day is invalid for the month you have chosen, it will be trimmed to the latest valid day,<br>
+  e.g. `31/02/2021` will be edited to `28/02/2021` by CHIM.
+
 </div>
 
-### Adding an order: `addorder`
+### Viewing help : `help`
 
-Adds an order to CHIM.
+Shows a message explaning how to access the help page.
 
-Format: `addorder t/CHEESE_TYPE q/QUANTITY p/PHONE_NUMBER [d/ORDER_DATE]`
-* The specified `QUANTITY` must be a positive integer.
-* The specified `PHONE_NUMBER` must belong to an existing user.
+![help message](images/helpMessage_CHIM.png)
 
-Example: `addorder t/Parmesan q/2 p/65555555`
+Format: `help`
 
-### Adding a cheese: `addcheese`
 
-Adds a cheese to CHIM.
+### Features for Customers
 
-Format: `addcheese t/CHEESE_TYPE q/QUANTITY [d/MANUFACTURE_DATE] [d/EXPIRY_DATE]`
-* The specified `QUANTITY` must be a positive integer.
+#### Listing customers: `listcustomers`
+
+Displays the customers in CHIM.
 
-Example: `addcheese t/Parmesan q/2`
+Format: `listcustomers`
 
-### Adding a customer: `addcustomer`
+#### Adding a customer: `addcustomer`
 
 Adds a customer to CHIM.
 
-Format: `addcustomer n/CUSTOMER_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/tags...]`
+Format: `addcustomer n/CUSTOMER_NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG…]​`
 
-Example: `addcustomer n/John Doe p/65555555 e/johndoe@gmail.com a/Blk 436 Serangoon Gardens St 26 #01-01`
+Example: `addcustomer n/John Doe p/61234567 e/johndoe@gmail.com a/Blk 436 Serangoon Gardens St 26 #01-01`
 
-### Editing an order: `editorder`
+#### Editing a customer: `editcustomer`
 
-Edits an existing uncompleted order in CHIM.
+Edits an existing customer in CHIM.
 
-Format: `editorder INDEX [t/CHEESE_TYPE] [q/QUANTITY] [p/PHONE_NUMBER] [d/ORDER_DATE]`
-* The specified `INDEX` and `QUANTITY` must be a positive integer.
-* The specified `PHONE_NUMBER` must belong to an existing user.
+Format: `editcustomer INDEX [n/CUSTOMER_NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG…]`
+* At least one of the optional fields must be provided to edit the customer.
 
-Example: `editorder 2 t/Parmesan q/2 p/65555555`
+Example: `editcustomer 4 n/Jane Lim p/65558888`
 
-### Editing a cheese: `editcheese`
+#### Deleting a customer: `deletecustomer`
 
-Edits an existing unassigned cheese in CHIM.
+Deletes a customer from the list of customers, using their phone number as an identifier.
 
-Format: `editcheese INDEX [t/CHEESE_TYPE] [d/MANUFACTURE_DATE] [d/EXPIRY_DATE]`
-* The specified `INDEX` must be a positive integer.
+If the customer has orders, their orders will be deleted by CHIM.
```

Comment:
WIll be good to mention that if there are some orders completed , the cheeses assigned will be deleted as well.